plugins/Goals/Columns/Metrics/GoalSpecificProcessedMetric.php

```diff
@@ -60,6 +62,16 @@ protected function getGoalMetrics(Row $row)
             $alternateKey = 'idgoal=' . $this->idGoal;
             if (isset($allGoalMetrics[$alternateKey])) {
                 return $allGoalMetrics[$alternateKey];
+            } elseif ($this->idGoal === Piwik::LABEL_ID_GOAL_IS_ECOMMERCE_ORDER) {
```

Comment:
fixes ui tests as sometimes we have a string here and sometimes an integer / metric id. Wasn't needed before but now